adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -169,18 +179,28 @@ class GFF3Parser extends FeatureParser {
  * In lieu of a specification, see:
  * https://samtools.github.io/htsjdk/javadoc/htsjdk/htsjdk/samtools/util/IntervalList.html
  */
-class IntervalListParser extends Serializable {
-  def parse(line: String): (Option[SequenceRecord], Option[Feature]) = {
+private[rdd] class IntervalListParser extends Serializable with Logging {
```

Comment:
I assume this was meant to be on a different line? That change tracks all the way back to https://github.com/bigdatagenomics/adam/pull/686. I'd want to get @ryan-williams' thoughts on that code before we yanked it out or changed it. I think we're getting into a debate that's somewhat orthogonal to this PR, which is _what_ exactly the `.interval_list` format is. I've just added code to said parser so that it passes round trip tests. If we'd like to change the parser, I think we should open an issue and discuss that there, and not block this PR on it. That's just my opinion though. Perhaps let's discuss on the call tomorrow?